src/components/Contacts/ContactDetails/ContactDetailsTab/Other/EditContactOtherModal/EditContactOtherModal.tsx

```diff
@@ -436,16 +434,21 @@ export const EditContactOtherModal: React.FC<EditContactOtherModalProps> = ({
                           {languages.map(
                             (value) =>
                               value?.value && (
```

Comment:
Honestly, we should probably check that `value.id` is set. We don't want `MenuItem`s with an `undefined` key. And value isn't falsy so `value.id && value.value`  is sufficient.